docs/dev/development.adoc

```diff
@@ -9,10 +9,21 @@ toc::[]
 
 Requires *Go 1.12*
 
-**WARNING**: If you are adding any features that require a higher version of golang, such as golang 1.13
-for example, please contact maintainers to check of the releasing systems can handle the newer versions.
+Testing and release builds happen with the above version. Developers are advised to stick to this version if they can but it is not compulsory.
 
-If that is ok, please ensure you update the required golang version, both here and in the file link:/scripts/rpm-prepare.sh[`scripts/rpm-prepare.sh`]
+
+[WARNING]
+====
+If you are adding any features that require a higher version of golang, that the one mentioned above, please contact maintainers to check of the releasing systems can handle the newer versions. If that is ok, please ensure you update the required golang version, both here and in the files below, in your PR.
```

Comment:
`than the one mentioned above`

Reply:
`please contact the maintainers`

Reply:
`please contact the maintainers in order to check if the releasing systems can handle the newer versions`

Reply:
fixed